### ISSUE TITLE
Implement an API feature to request the TM Database

### ISSUE THREAD

**Issue by pierzen** [hotosm/osm-tasking-manager#177]
To monitor major activations, we experimented recently using overview maps where imagery layers and TM layers can be added
see
http://umap.openstreetmap.fr/fr/map/hot-yolanda-haiyan-typhoon-activation_3628#7/11.738/123.563
http://umap.openstreetmap.fr/en/map/central-african-republic_3868

GeoJson Export from the TM providing outiline of the Job could be used as dynamic url in tools such as uMap and similar tools.

Infos about individual tasks (ie. user, status, coordinates) could also be used to monitor. For example, it could be possible to extract contributor statistics for jobs completed. This again would help to monitor TM jobs.

**Comment by mikelmaron** [comment]
Good idea. A complete API would also allow creation of Jobs. The use case would be for other tools to automatically create Jobs, without a manual step.
